logger/sentry.go

```diff
@@ -0,0 +1,122 @@
+package logger
+
+import (
+	"fmt"
+
+	"github.com/getsentry/sentry-go"
+)
+
+// A SentryLogger
+type SentryLogger struct {
+	l Logger
+	//ctx LogContext
+}
+
+// NewSentryLogger constructs a SentryLogger based off the provided TrailsLogger.
+func NewSentryLogger(tl *TrailsLogger, dsn string) Logger {
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn:          dsn,
+		Environment:  tl.env,
+		IgnoreErrors: []string{"write: broken pipe"},
```

Comment:
Make this configurable? I didn't want broken pipe errors on archive given how much networking I'm doing, but I don't think you'd want that elsewhere.